### ISSUE TITLE
CPU usage spike when mining

### ISSUE THREAD

**Issue by ThePangolin** [capSAR273/TheWastes#14]
I've noticed whenever I start mining my CPU usage spikes from 0-3% to around 30-40%, and it drops back down whenever I stop mining. Don't know if this counts as a bug, but it sounds like it isn't supposed to happen.

**Comment by capSAR273** [comment]
The way the program works when mining, is it is rapidly refreshing the screen to make the mining animation work. This requires the program to re-display the text on screen rapidly, which is probably why the CPU usage is increased during mining.

**Comment by Huw2k8** [comment]
Yeah I think capSAR's got it, mining is shit at the moment and will be sorted out later without the screen flip
